### DISCUSSION TITLE
Long term updates

### DISCUSSION THREAD

**Discussion by gonzaloalbito** [Desuuuu/DGUS-reloaded-Klipper#22] [General]
Hi! I'll ask about long term plans to upgrade this repository code with latest klipper changes. Recently I bought a SKR mini E3 V3 which comes with an STM32G0B1 so I needed to build klipper from the Bigtreetech fork and branch (https://github.com/bigtreetech/Klipper/tree/SKR-mini-E3-V3.0-G0B1) which will eventually be merged in the klipper repository (i hope). 
I tried to clone their repository and merge your changes on it but i hane nbot enought klipper knowledges and time to do it on my own, so I'm without screen for now (i hope).
 So I was thinking about which are this project upgrade plans. Will it be updated with latest klipper changes in any moment? If so, I cross my fingers for my board being officially supported then.
Kind regards and nice work

**Comment by Desuuuu** [comment]
STM32G0 should now be supported on the `dgus-display` branch (details [here](https://github.com/Desuuuu/klipper/discussions/74)).

The `dgus-reloaded` branch (previously `master`) is also up-to-date but I haven't updated anything related to the newly supported MCUs, so I don't know whether or not it will work on STM32G0.
